src/gfx/shaders/Uber.frag

```diff
@@ -560,17 +560,14 @@ void main() {
   #ifdef NORMALS_TO_G_BUFFER
     #if defined (SPHERE_SPRITE) || defined (CYLINDER_SPRITE)
       vec3 viewNormaInColor = viewNormalSprites;
+      float frontFaced = 1.0;
     #else
       vec3 viewNormaInColor = viewNormal;
-    #endif
-    flipNormal = 1.0;
-    // invert normals on invisible front faces to all of them been directed to the side of viewer
-    #ifdef DOUBLE_SIDED_G_BUFFER
-      flipNormal = normalSign;
+      float frontFaced = normalSign;
     #endif
     // [-1, 1] -> [0, 1]
-    viewNormaInColor = 0.5 * viewNormaInColor * flipNormal + 0.5;
-    gl_FragData[1] = vec4(viewNormaInColor, 1.0);
+    viewNormaInColor = 0.5 * viewNormaInColor + 0.5;
+    gl_FragData[1] = vec4(viewNormaInColor, frontFaced);
```

Comment:
Hi, thank you for the change. There are a few comments from me. Please follow them, and we accept you request with pleasure.

Reply:
Thanks for you comments. I fix code and hope that it becomes better